FreeRTOS/FreeRTOS-Plus-TCP
# Update FreeRTOS_debug_printf in comments

Update FreeRTOS_debug_printf in [line 880
](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/source/FreeRTOS_Routing.c#L880)
Description
-----------
In [line 855](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/source/FreeRTOS_Routing.c#L855), the #if directive is checking for the "ipconfigHAS_PRINTF" conditional expression. Following this conditional check, [line 872](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/source/FreeRTOS_Routing.c#L872) it has "FreeRTOS_debug_printf" which is a mismatch, hence updating the comment. It also fixes the "variable 'xRetNtopFrom' set but not used" compilation warning

Test Steps
-----------
This is just a print replacement

Checklist:
----------
Not Available

Related Issue
-----------
Not an issue


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
